src/components/LibraryListForm/index.tsx

```diff
@@ -104,16 +103,25 @@ const LibraryListForm = ({ libraryList = [] }: LibraryListFormProps) => {
   );
 
   return (
-    <Autosuggest
-      suggestions={suggestions}
-      onSuggestionsFetchRequested={onSuggestionsFetchRequested}
-      onSuggestionsClearRequested={onSuggestionsClearRequested}
-      getSuggestionValue={getSuggestionValue}
-      renderSuggestion={renderSuggestion}
-      inputProps={inputProps}
-      renderInputComponent={renderInputComponent}
-      renderSuggestionsContainer={renderSuggestionsContainer}
-    />
+    <>
+      <h3>Home Library</h3>
+      <p>
+        Choosing a home library will help us make sure you&apos;re getting
+        everything you need from a branch that&apos;s most convenient for you.
+        You can change your home library at any point through your account.
+      </p>
+      <p>Skipping this step will automatically a home library.</p>
```

Comment:
Missing word ("choose"?) after "automatically"

Reply:
Yes, good catch. It's "default".